dev/bots/analyze.dart

```diff
@@ -1772,7 +1772,11 @@ Future<void> _checkConsumerDependencies() async {
 
       final List<String> currentDependencies = (currentPackage['dependencies']! as List<Object?>).cast<String>();
       for (final String dependency in currentDependencies) {
-        workset.add(dependencyTree[dependency]!);
+        // Don't add dependencies we've already seen or we will get stuck
```

Comment:
Can you link to https://github.com/dart-lang/test/issues/1979 with a TODO to re-evaluate this once that issue is closed?